### PR TITLE
[@types/bootstrap] carousel option is called ride

### PR DESCRIPTION
**Pull request by darthf1**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://getbootstrap.com/docs/5.1/components/carousel/#options
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/bootstrap/js/dist/carousel.d.ts [hunk 1]
```diff
@@ -98,7 +98,7 @@ declare namespace Carousel {
          *
          * @default false
          */
-        slide: string | boolean;
+        ride: string | boolean;
```
Comment (peterblazejewicz) [inline types/bootstrap/js/dist/carousel.d.ts:101]:
```suggestion
        ride: 'carousel' | boolean;
```
can we update this as well? (there is no other options support, it reflects data attrib configuration)

**Comment by martin-badin** [comment]
The URL to documentation in the description is wrong.

File: types/bootstrap/js/dist/carousel.d.ts (hunk 1 above)
Comment (darthf1) [reply types/bootstrap/js/dist/carousel.d.ts]: I also changed the values for `pause`. Not 100% sure, but looks like this if you read the docs

**Comment by darthf1** [comment]
> The URL to documentation in the description is wrong. 

I removed the property entirely as it wasn't used

**Comment by martin-badin** [comment]
The URL here is wrong

_Provide a URL to documentation or source code which provides context for the suggested changes: https://getbootstrap.com/docs/5.1/components/dropdowns/#options_

**Comment by darthf1** [comment]
Ah! my bad.

File: types/bootstrap/js/dist/carousel.d.ts [hunk 2]
```diff
@@ -90,15 +83,15 @@ declare namespace Carousel {
...
+        pause: 'hover' | false;
 
         /**
          * Autoplays the carousel after the user manually cycles the first item. If
          * "carousel", autoplays the carousel on load.
          *
          * @default false
          */
-        slide: string | boolean;
+        ride: 'carousel' | false;
```
Comment (peterblazejewicz) [inline types/bootstrap/js/dist/carousel.d.ts:94]:
```suggestion
        ride: 'carousel' | boolean;
```
should be probably left as `boolean`, as the code uses expression evaluation:
https://github.com/twbs/bootstrap/blob/main/js/src/carousel.js#L463

**Review by martin-badin — CHANGES_REQUESTED** [review]

**Review by martin-badin — APPROVED** [review]
Looks good

**Comment by darthf1** [comment]
Ready to merge

**Pull request merged**